torchtext/data/datasets_example_doc.py

```diff
@@ -0,0 +1,51 @@
+    {"dataset_name": "UDPOS", "split": "valid", "first_line": [["From", "the", "AP"], ["ADP", "DET", "PROPN"], ["IN", "DT", "NNP"]]},
+    {"dataset_name": "UDPOS", "split": "test", "first_line": [["What", "if", "Google"], ["PRON", "SCONJ", "PROPN"], ["WP", "IN", "NNP"]]},
+    {"dataset_name": "CoNLL2000Chunking", "split": "train", "first_line": [["Confidence", "in", "the"], ["NN", "IN", "DT"], ["B-NP", "B-PP", "B-NP"]]},
+    {"dataset_name": "CoNLL2000Chunking", "split": "test", "first_line": [["Rockwell", "International", "Corp."], ["NNP", "NNP", "NNP"], ["B-NP", "I-NP", "I-NP"]]},
+    {"dataset_name": "Multi30k", "split": "train", "first_line": ["Zwei junge wei\u00dfe M\u00e4nner sind im Freien in der N\u00e4he vieler B\u00fcsche.\n", "Two young, White"]},
+    {"dataset_name": "Multi30k", "split": "valid", "first_line": ["Eine Gruppe von M\u00e4nnern l\u00e4dt Baumwolle auf einen Lastwagen\n", "A group of men are loading cotton onto a truck\n"]},
+    {"dataset_name": "Multi30k", "split": "test", "first_line": ["Ein Mann mit einem orangefarbenen Hut, der etwas anstarrt.\n", "A man in an orange hat starring at something.\n"]},
+    {"dataset_name": "IWSLT2016", "split": "train", "first_line": ["David Gallo: Das ist Bill Lange. Ich bin Dave Gallo.\n", "David Gallo: This is Bill Lange. I'm Dave Gallo.\n"]},
+    {"dataset_name": "IWSLT2016", "split": "valid", "first_line": ["Als ich 11 Jahre alt war, wurde ich eines Morgens von den Kl\u00e4ngen heller Freude geweckt.\n", "When I was 11"]},
+    {"dataset_name": "IWSLT2016", "split": "test", "first_line": ["Als ich in meinen 20ern war, hatte ich meine erste Psychotherapie-Patientin.\n", "When I was in my 20s, I saw my"]},
+    {"dataset_name": "IWSLT2017", "split": "train", "first_line": ["Vielen Dank, Chris.\n", "Thank you so much, Chris.\n"]},
+    {"dataset_name": "IWSLT2017", "split": "valid", "first_line": ["Letztes Jahr habe ich diese beiden Folien gezeigt, um zu veranschaulichen, dass die arktische Eiskappe, die"]},  # noqa: E231
+    {"dataset_name": "IWSLT2017", "split": "test", "first_line": ["Vor einigen Jahren, hier bei TED, stellte Peter Skillman einen Design-Wettbewerb namens \"Die Marshmallow-Herausforderung"]},  # noqa: E231
+    {"dataset_name": "WMT14", "split": "train", "first_line": ["Wiederaufnahme der Sitzungsperiode\n", "Res@@ um@@ ption of the session\n"]},
+    {"dataset_name": "WMT14", "split": "valid", "first_line": ["Eine repub@@ li@@ kanische Strategie , um der Wieder@@ wahl von Obama entgegen@@ zu@@ treten\n", "A Republic@@ an strategy"]},
+    {"dataset_name": "WMT14", "split": "test", "first_line": ["Gut@@ ach : Noch mehr Sicherheit f\u00fcr Fu\u00dfg\u00e4n@@ ger\n", "Gut@@ ach : Incre@@ ased safety for pedestri@@ ans\n"]},
+    {"dataset_name": "WikiText2", "split": "train", "first_line": "This is an example for WikiText2 dataset"},
```

Comment:
The first line of wikitext2 is empty, so this might cause misleading documentation. I'd instead update the documentation of wikitext2 to iterate until it finds a non-empty line.